custom_components/tahoma/sensor.py

```diff
@@ -63,6 +63,22 @@ def unit_of_measurement(self):
 
         return None
 
+    
+    @property
+    def device_class(self) -> Optional[str]:
+        """Return the device class of this entity if any."""
+
+        if self.tahoma_device.uiclass == "TemperatureSensor":
```

Comment:
Nice! Could you maybe add it to const.py like the others? You can find an example in the `binary_sensor.py`.

https://github.com/iMicknl/ha-tahoma/blob/master/custom_components/tahoma/const.py

Reply:
Do you know if the unit of measurement can be skipped when you set the device_class?